site/Gemfile.lock

```diff
@@ -161,4 +161,4 @@ DEPENDENCIES
   tzinfo-data
 
 BUNDLED WITH
-   1.17.2
+   2.2.21
```

Comment:
Could you please comment why we need this change?

Reply:
It happend after the build.sh. :man_shrugging: I can change to the old version. What do u think?

Reply:
Yeah might be better/

Reply:
done